src/dbnode/storage/index/compaction/compactor.go

```diff
@@ -0,0 +1,358 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
```

Comment:
2019

Reply:
As talked, going to leave these as a fair few were from 2018 hah.